serverless/knative_eventing/serverless-knative-eventing.adoc

```diff
@@ -10,4 +10,10 @@ Event-driven architecture is based on the concept of decoupled relationships bet
 
 For more information about event-driven architecture, see link:https://www.redhat.com/en/topics/integration/what-is-event-driven-architecture[What is event-driven architecture?]
 
+In a Knative Eventing workflow, event producers (for example, a Kafka cluster or the Kubernetes API Server) send information about changes in the state of the system to an event source.
```

Comment:
This sentence is pretty long. Can you break it up?
s/information about changes in the state of the system to an event source/send information to an event source about changes to system state./